docs/classic-ui/portlets.md

```diff
@@ -1,13 +1,185 @@
 ---
 myst:
   html_meta:
-    "description": ""
-    "property=og:description": ""
-    "property=og:title": ""
-    "keywords": ""
+    "description": "Basic information about portlets in classic UI"
+    "property=og:description": "Basic information about portlets in classic UI"
+    "property=og:title": "Portlets"
+    "keywords": "portlets"
 ---
 
 (classic-ui-portlets-label)=
 
 # Portlets
 
+## What is a Portlet?
+
+In Plone, a portlet is a small, modular piece of content that can be displayed in a specific area of a web page. Portlets are typically used to display information that is relevant to the current context, such as the latest news, upcoming events, or a list of related documents.
+
+The context is either the current part of the content hierarchy, the current user's group memberships, or the current content type.
+Thus, if a portlet was set on a folder, all contained items do display the portlet unless it is explicitly blocked.
```

Comment:
I've read this sentence three times, and it still does not make sense to me. This needs some clarity.

Reply:
I refined this part. Better now?

Reply:
Waaaaay better! Inheritance rocks. Thank you!